src/ship.py

```diff
@@ -12,15 +12,15 @@ def __init__(self, x, y):
         self.hit = False
 
 class Ship:
-    def __init__(self):
+    def __init__(self): #W3 defn: "A reference to the current instance of the class, and is used to access variables that belongs to the class."
```

Comment:
This is a standard Python thing that applies to all classes so shouldn't be commented here